packages/manager/apps/container/src/container/legacy/account-sidebar/Shortcuts/Tile/Tile.tsx

```diff
@@ -28,22 +30,25 @@ const Tile = ({ shortcut = {} as Shortcut }: Props): JSX.Element => {
   };
 
   return (
-    <div className={cssClassName}>
-      <a
-        href={shortcut.url}
-        target="_top"
-        onClick={() => handleShortcutClick(shortcut)}
+
+    <div className="m-1">
+    <a
+      href={shortcut.url}
+      target="_top"
+      onClick={() => handleShortcutClick(shortcut)} className="block"
+    >
+      <OsdsTile hoverable variant={ODS_TILE_VARIANT.stroked} color={ODS_THEME_COLOR_INTENT.primary}>{shortcut.icon}</OsdsTile>
+      <OsdsText
+        color={ODS_THEME_COLOR_INTENT.text}
+        level={ODS_TEXT_LEVEL.body}
+        size={ODS_TEXT_SIZE._200}
+        className='font-bold'
```

Comment:
You should install the prettier plugin in your vscode to have the correct formatting